html/php-pre-challenge3/index.php

```diff
@@ -1,6 +1,58 @@
 <?php
 $limit = $_GET['target'];
-
+$limit = mb_convert_kana($limit, 'a' , 'UTF-8');
+if(!is_numeric($limit) || $limit <= 0 || preg_match('/^([1-9]\d*|0)\.(\d+)?$/', $limit)) {
+    http_response_code(400);
```

Comment:
http_response_code()はヘッダを設定しているだけでスクリプト自体は止まらないのでexit()を入れましょう

```suggestion
    http_response_code(400);
    exit();
```